src/test/java/com/epam/tc/hw4/utils/AssertStep.java

```diff
@@ -0,0 +1,122 @@
+package com.epam.tc.hw4.utils;
+
+import static com.epam.tc.hw4.tests.BaseTest.driver;
+import static org.assertj.core.api.Assertions.assertThat;
+import static pages.Header.DIFF_ELEMENTS_TITLE;
+import static pages.Header.HOME_PAGE_TITLE;
+import static pages.Header.headerNamesExpected;
+import static pages.Header.userNameLogged;
+import static pages.IndexPage.expectedText;
+import static pages.LeftSection.names;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.asserts.SoftAssert;
+import pages.DiffElementsPage;
+import pages.Header;
+import pages.IndexPage;
+import pages.LeftSection;
+
+public class AssertStep extends PageObject {
+    SoftAssert softAssert = new SoftAssert();
+    private static final String iFrameElement = "frame";
```

Comment:
static final variables of class in jcc named in upper case, for ex. FRAME_ELEMENT